modules/installation-two-node-cluster-with-fencing.adoc

```diff
@@ -0,0 +1,47 @@
+:_mod-docs-content-type: CONCEPT
+[id="sample-install-config-two-node-fencing_{context}"]
+= Sample install-config.yaml for a two-node OpenShift cluster with fencing
+
+You can use the following `install-config.yaml` as a template for deploying a two-node OpenShift cluster with fencing.
+
+.Sample install-config.yaml for installing a two-node OpenShift cluster with fencing
+[source,yaml]
```

Comment:
we need a UPI example as well. It actually doesn't make sense that we provide an IPI example in the UPI section of the docs